aten/src/ATen/native/cuda/jit_utils.cpp

```diff
@@ -1437,7 +1437,7 @@ std::optional<std::string> get_cache_dir() {
   const char* p_cache_dir = cache_dir.c_str();
   const bool cache_dir_exists = (access(p_cache_dir, F_OK) == 0);
   if (!cache_dir_exists) {
-    std::string s_ptkcp = std::string(ptkcp);
+    std::string s_ptkcp = ptkcp.value();
```

Comment:
I can't easily tell if it is guaranteed not to be nullopt here

Reply:
It must be